robot/Cumulus/tests/browser/gift_entry/missing_permissions.robot

```diff
@@ -0,0 +1,153 @@
+  Go to Page                  Landing    GE_Gift_Entry  default=permissions_error
+  Current Page Should Be      Landing    GE_Gift_Entry  default=permissions_error
+  Page Should Contain         You must have permission to edit the following fields: ${NS}${di_batch}: (${NS}${field})      
+  Change Field Permissions    add  ${di_batch}  ${field}  ${perm_set_name}
+  Reload Page
+  Page Should Not Contain     You must have permission to edit the following fields: ${NS}${di_batch}: (${NS}${field})
+
+
+Verify Permissions Error When Form Template Description Field Access is Revoked
+  [Documentation]          Disables field permissions for the Description field of the Form Template Object in the Gift_Entry_Perms permission set,
+  ...                      navigates to the Gift Entry app to verify the permissions error message, then re-adds the object
+  ...                      permissions and reloads the Gift Entry app to verify the permissions error no longer displays.
+  [tags]                   unstable      feature:GE        W-8279499
+  [Teardown]               Run Keywords  Field Permissions Cleanup  ${form_temp}  ${field}  ${perm_set_name}
+  ...                      AND  Delete All Cookies
+  ...                      AND  Close All Browsers
+
+  ${field}  Set Variable      Description__c
+  Change Field Permissions    remove  ${form_temp}  ${field}  ${perm_set_name}
+  Open Test Browser           alias=otherfour  useralias=${user_alias}                     
+  Go to Page                  Landing    GE_Gift_Entry
+  Current Page Should Be      Landing    GE_Gift_Entry
+  Click Link                  Templates
+  Page Should Contain         You must have permission to edit the following fields: ${NS}${form_temp}: (${NS}${field})      
+  Change Field Permissions    add  ${form_temp}  ${field}  ${perm_set_name}
+  Reload Page
+  Click Link                  Templates
+  Page Should Not Contain     You must have permission to edit the following fields: ${NS}${form_temp}: (${NS}${field})
+
+
+Verify Target Object Field Without Access Displays Permissions Errors
+  [Documentation]          Disables object permissions for the Check/Reference field in the custom Payment
+  ...                      Object in the Gift_Entry_Perms permission set, then navigates to the GE form to validate that
+  ...                      the permissions error displays on the page. The test then navigates to the Template Builder to
+  ...                      verify that the error messages display for the fields without access. Then, permissions are
+  ...                      restored to the field, and then the test navigates back to the GE form, and then to the Template
+  ...                      Builder to ensure the permissions error messages no longer display.
+  [tags]                   unstable      feature:GE        W-8279499
+  [Teardown]               Run Keywords  Field Permissions Cleanup  ${obj}  ${field}  ${perm_set_name}
```

Comment:
As part of teardown are you resetting everything back?

Reply:
Yes. It happens on a per-test basis, so even if you Ctrl+C the test mid-run, it all gets reset.